clients/client-py/release.sh

```diff
@@ -26,6 +26,7 @@ rm -rf build/
 if ! command -v uv &> /dev/null; then
     echo "uv not found, installing..."
     curl -LsSf https://astral.sh/uv/install.sh | sh
+    source $HOME/.local/bin/env
```

Comment:
I think this works, because it would be extremely unusual for `$HOME` to contain white space characters, but `"${HOME}/.local/bin/env"` would be safer (i.e. with quotation marks around the expression).